static/js/GuideBox.jsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Component from 'react-class';
+import Sefaria from "./sefaria/sefaria";
+import TextRange from "./TextRange";
+import {AiInfoTooltip} from './Misc';
+
+// Constants for different prompt states
+const QUESTIONS = "questions";
+const SUMMARIES = "summaries";
+const COMMENTARIES = "commentaries";
+
+/**
+ * Renders the question prompts in a box.
+ * @param {Array} props.prompt - The array of question prompts.
+ * @param {Function} props.onClick - The function to handle click events on prompts.
+ * @returns {JSX.Element} The QuestionBox component.
+ */
+const QuestionBox = ({ prompt, onClick }) => {
+
+  const questionHelperText = (p) => {
+    return `${p.commentaries.length} ${p.commentaries.length > 0 ? 'Answers' : 'Answer'}`;
+  }
+
+  return (
+    <div>
+      <div className="guideTitle">
+        Key Questions
+      </div>
+      {prompt.map((p, i) => {
+        return (
+          <div key={i} className="guidePromptBox" onClick={() => onClick(p, i)}>
+            <p>{p.question}</p>
+            <span>{questionHelperText(p)}</span>
+          </div>
+        )
+      })}
+    </div>
+  );
+}
+
+/**
+ * Renders the summary prompts in a box.
+ * @param {Object} props.prompt - The summary prompt object.
+ * @param {Function} props.onClick - The function to handle click events on prompts.
+ * @returns {JSX.Element} The SummaryBox component.
+ */
+const SummaryBox = ({ prompt, onClick }) => {
+  const title = (commentaryRef) => {
+    const index = Sefaria.parseRef(commentaryRef)?.index;
+    return index ? index.replace(/ on Pesach Haggadah$/, "") : commentaryRef;
```

Comment:
@EliezerIsrael this was done bespokely for the Haggadah project. It solved an issue with showing the names of the authors properly. This is clearly not a generic solution and no longer applicable, should we determine a new product definition for what we show here (for instance, show the name of the index no matter what) or have something more generic (like use something other than the index to get the correct data)

Reply:
Fixed, and added a :"todo" for generic replacement logic